keps/sig-node/127-user-namespaces/README.md

```diff
@@ -765,13 +785,27 @@ will rollout across nodes.
 
 ###### What specific metrics should inform a rollback?
 
+On Kubernetes side, the kubelet should start correctly.
+
+On the node runtime side, a pod created with pod.spec.HostUsers=false should be on RUNNING state if
+all node requirements are met.
```

Comment:
I'm not a fan of this answer - if I'm managing 1000s of clusters, it's not something I can reasonably monitor. Can we rely on easier to monitor conditions - ideally some metrics?

Reply:
We can add metrics to signal on the errors listed here in the KEP (in the section "What are other known failure modes?"). 

This can potentially cover a lot at the kubelet layer. But things like one of the the filesystems used by the pod doesn't support idmap mounts on the kernel running will not be possible to detect from the kubelet. The runtime will return a clear error on that case, that is shown to the user, but I don't think we can have a metric for that.

Does doing that sounds like a good plan for you?

I'm not convinced about this, though. All errors are returned to the user, so it's very visible if it fails. And having a proliferation of metrics can also harm, especially long-term. Do you think this is definitely a "yes, this metrics are useful" or do you have mixed feelings about it too?

Also, do you think this is needed for beta graduation or can be added while the feature is in beta and still disabled by default?